### DISCUSSION TITLE
Why there is no paragraph in the response?

### DISCUSSION THREAD

**Discussion by maketesteasier** [deepgram/community#281] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

Usually in a response I should get paragraph, but with this audio file: 
I failed to get it as seen in the following picture: 

Also, I tried to call your api from google colab, using that audio file as well, but I get this: 

Requesting transcript...
Your file may take up to a couple minutes to process.
While you wait, did you know that Deepgram accepts over 40 audio file formats? Even MP4s.
To learn more about customizing your transcripts check out developers.deepgram.com
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/deepgram/_utils.py](https://localhost:8080/#) in attempt()
    145         try:
--> 146             with urllib.request.urlopen(req, timeout=timeout) as resp:
    147                 content = resp.read().strip()

11 frames
HTTPError: HTTP Error 411: Length Required

During handling of the above exception, another exception occurred:

Exception                                 Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/deepgram/_utils.py](https://localhost:8080/#) in attempt()
    153                 return body
    154         except urllib.error.HTTPError as exc:
--> 155             raise (Exception(f'DG: {exc}') if exc.code < 500 else exc)
    156 
    157     tries = RETRY_COUNT

Exception: DG: HTTP Error 411: Length Required

**Comment by liechtym** [comment]
I believe this is a duplicate of the issue I posted here: https://github.com/orgs/deepgram/discussions/273 . If so, please upvote it so it will receive attention from their team.

**Comment by jkroll-deepgram** [comment]
Hi @maketesteasier, @liechtym is correct that this same issue was first reported yesterday. I will provide general updates on that thread.

I was not able to replicate this with your audio. Do you have a request ID for one of the requests that did not return paragraphs? That will enable us to look into your specific details.

Edit: I see your request ID from the other thread, thanks. For reference: 1daba5b4-443c-4736-b19b-67d9aa6ffd71

**Reply by maketesteasier** [reply]
Thanks so much. I will keep waiting on this page for your update.
